crypto/x509/x509_req.c

```diff
@@ -185,6 +185,9 @@ int X509_REQ_get_attr_by_OBJ(const X509_REQ *req, const ASN1_OBJECT *obj,
   for (; lastpos < n; lastpos++) {
     const X509_ATTRIBUTE *attr =
         sk_X509_ATTRIBUTE_value(req->req_info->attributes, lastpos);
+    if (attr == NULL) {
```

Comment:
I'm not sure if this can happen, since n was set the number of attributes, if I understood correctly. There is a similar code here https://github.com/aws/aws-lc/blob/8402a6eecfff195793a57b5bf1cf371f1218da3c/crypto/x509/t_req.c#L153-L155

Reply:
yeah, I had the same thought but I'm not 100% certain it can't happen. The additional check is very cheap anyway, so no harm in having it.